cluster/ubuntu/util.sh

```diff
@@ -14,55 +14,65 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# A library of helper functions that each provider hosting Kubernetes must implement to use cluster/kube-*.sh scripts.
+# A library of helper functions that each provider hosting Kubernetes must
+# implement to use cluster/kube-*.sh scripts.
 set -e
 
-SSH_OPTS="-oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null -oLogLevel=ERROR"
+MASTER=
+MASTER_IP=
+NODE_IPS=
 
-MASTER=""
-MASTER_IP=""
-NODE_IPS=""
+
+BASE_DIR="$KUBE_ROOT/cluster/ubuntu"
+
+SSH_OPTS="\
+ -oStrictHostKeyChecking=no\
+ -oUserKnownHostsFile=/dev/null\
+ -oLogLevel=ERROR\
+"
 
 # Assumed Vars:
 #   KUBE_ROOT
 function test-build-release {
   # Make a release
-  "${KUBE_ROOT}/build/release.sh"
+  "$KUBE_ROOT/build/release.sh"
```

Comment:
Why? There are no expansions that require them In all my replacements, the curly braces are not needed at all. I've only left them in the expansions and arrays where they are required.